### PR TITLE
Medium scraper

### LINKED ISSUE #1040: Medium Articles Scrapper
# Aim

Scrap articles from Medium

# Details

Come up with a script that scraps `n` number of articles from `any category` such as Python, JS, Android provided by the user and stores all the `n` articles separately in PDFs. The PDF file should contain the title, article content, and author-name

User Input: `n` and `category`
Expected Output: n PDF files

### PR DESCRIPTION
**Pull request by Ayushjain2205**
# Description

Script to scrape articles from medium depending on user input

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] I have added the project meta data in the PR template.
- [x] I have created the ``requirements.txt`` file if needed.

## Project Metadata

`` If there is no-file/nothing to fill the below fields with, then type: none ``

`` Example: `` If no requirements.txt needed/present, then type none in ``Requirments``.

Category:
- [ ] Calculators
- [ ] AI/ML
- [x] Scrappers
- [ ] Social_Media
- [ ] PDF
- [ ] Image_Processing
- [ ] Video_Processing
- [ ] Games
- [ ] Networking
- [ ] OS_Utilities
- [ ] Automation
- [ ] Cryptography
- [ ] Computer_Vision
- [ ] Fun
- [ ] Others

Title: Medium Scraper

Folder: Medium-Scraper

Requirements: requirements.txt

Script: scraper.py

Arguments: none

Contributor: Ayushjain2205

Description: Script to scrape articles from medium.com

### CODE REVIEW

File: Medium-Scraper/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Dev.to Scrapper
```
Comment (kaustubhgupta) [inline Medium-Scraper/README.md:1]: Umm, Maybe it's Medium? 😆

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: Medium-Scraper/README.md (hunk 1 above)
Comment (Ayushjain2205) [reply Medium-Scraper/README.md]: Oops my bad! resolved it now!

**Review by kaustubhgupta — APPROVED** [review]

**Review by santushtisharma10 — APPROVED** [review]
@Ayushjain2205 💯

**Review by Kushal997-das — APPROVED** [review]
LGTM 👍

**Review by avinashkranjan — APPROVED** [review]
LGTM

**Pull request merged**